src/admin-views/order_modifiers/coupon-edit.php

```diff
@@ -29,6 +29,8 @@
 	$heading = __( 'New Coupon', 'event-tickets' );
 }
 
+$limit_error_text = __( 'If setting a limit for a Coupon, it must be a positive number. Use 0 or leave empty for no limit.', 'event-tickets' );
```

Comment:
This is worded funny in my opinion.
```suggestion
$limit_error_text = __( 'Coupon Limit must be a positive number. Use 0 or leave empty for no limit.', 'event-tickets' );
```